tests/wpt/metadata/fetch/api/cors/cors-preflight-not-cors-safelisted.any.js.ini

```diff
@@ -1,60 +1,18 @@
 [cors-preflight-not-cors-safelisted.any.html]
-  [Need CORS-preflight for content-language/@ header]
-    expected: FAIL
-
   [Need CORS-preflight for content-language/\x01 header]
     expected: FAIL
```

Comment:
These tests still fail because of:
https://github.com/servo/servo/blob/a4bf7e51793813b2f5120c621a02771ad495be92/components/script/dom/headers.rs#L361-L363

Should we provide a PR to change HeaderValue?

Reply:
We should file an issue about it, at least.

Reply:
Done #27764 27764